src/main/java/org/alfresco/repo/search/impl/solr/SolrSQLHttpClient.java

```diff
@@ -200,6 +200,15 @@ public ResultSet executeQuery(SearchParameters searchParameters, String language
                 locales.put(I18NUtil.getLocale());
             }
             body.put("locales", locales);
+
+            // FIXME
```

Comment:
is this comment needed?

Reply:
No, not really. I've put the same comment in all MRs I've created as it was not intended as production ready code (hence the MRs being closed). This MR is created only for future reference.